frontend/app-development/layout/SettingsModalButton/SettingsModal/utils/tabUtils/accessControlTabUtils/accessControlTabUtils.ts

```diff
@@ -14,10 +14,10 @@ export const initialPartyTypes: PartyTypesAllowed = {
  * Map to handle the different text to display in the checkboxes
  */
 export const partyTypesAllowedMap: Record<AllowedPartyTypes, string> = {
-  bankruptcyEstate: 'settings_modal.access_control_tab_option_bankruptcy_estate',
-  organisation: 'settings_modal.access_control_tab_option_organisation',
   person: 'settings_modal.access_control_tab_option_person',
+  organisation: 'settings_modal.access_control_tab_option_organisation',
   subUnit: 'settings_modal.access_control_tab_option_sub_unit',
+  bankruptcyEstate: 'settings_modal.access_control_tab_option_bankruptcy_estate',
```

Comment:
Any reason these are moved?

Reply:
I just wanted to have the same order in Figma, but i can reorder it as they were. 😊

Reply:
Ah okei, understand! I dont know what is the best, but I think the original order was alphabetical, which might make more sense for an initial look. But I dont have any strong opinions about this 😊